pkg/server/event_handler_test.go

```diff
@@ -537,7 +535,8 @@ isOK: true`),
 }
 
 // Creates a fake http client.
-func newTestServer(handler func(w http.ResponseWriter, r *http.Request)) (*http.Client, func()) {
+func newTestServer(t *testing.T, handler func(w http.ResponseWriter, r *http.Request)) *http.Client {
```

Comment:
Nit: prefer accepting `testing.TB` (interface) instead of `*testing.T` (type). The interface makes it reusable for benchmarks.

Reply:
Done, thanks!